browser/resources/settings/brave_web3_domains_page/brave_web3_domains_page.ts

```diff
@@ -26,7 +26,6 @@ export class SettingBraveWeb3DomainsPageElement
 
   static get properties() {
     return {
-      showSnsRow_: Boolean,
```

Comment:
still referenced in `brave_web3_domains_page.html`

Reply:
Ah, missed it, thanks for catching this.
Fixed by https://github.com/brave/brave-core/pull/20898/commits/489e3989807691bf686cbe89e390e9b14e9cd5cb.